azure/resource_groups/app/template.json

```diff
@@ -119,24 +119,23 @@
 
     "applicationInsightsDeploymentName": "[concat(parameters('resourceNamePrefix'), '-application-insights')]",
     "appServiceCertificateDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-certificate')]",
+    "appServiceDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-web')]",
+    "appServiceWorkerDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-worker')]",
     "appServicePlanDeploymentName": "[concat(parameters('resourceNamePrefix'), '-app-service-plan')]",
+    "appServicePlanWorkerDeploymentName": "[concat(parameters('resourceNamePrefix'), '-worker-app-service-plan')]",
     "databaseServerFirewallRulesDeploymentName": "[concat(parameters('resourceNamePrefix'), '-database-server-firewall-rules')]",
     "databaseServerDeploymentName": "[concat(parameters('resourceNamePrefix'), '-database-server')]",
     "databaseDeploymentName": "[concat(parameters('resourceNamePrefix'), '-database')]",
     "storageAccountDeploymentName": "[concat(parameters('resourceNamePrefix'), '-storage-account')]",
     "vspDeploymentName": "[concat(parameters('resourceNamePrefix'), '-vsp')]",
-
     "storageAccountName": "[replace(concat(parameters('resourceNamePrefix'), 'storage'), '-', '')]",
-
     "databaseServerName": "[concat(parameters('resourceNamePrefix'), '-db')]",
-
     "applicationInsightsName": "[concat(parameters('resourceNamePrefix'), '-ai')]",
-
     "appServicePlanName": "[concat(parameters('resourceNamePrefix'), '-asp')]",
-
+    "appServicePlanWorkerName": "[concat(parameters('resourceNamePrefix'), '-wkr-asp')]",
     "appServiceName": "[concat(parameters('resourceNamePrefix'), '-as')]",
-    "appServiceRuntimeStack": "[concat('COMPOSE|', base64(parameters('appServiceDockerCompose')))]",
-
+    "appServiceWorkerName": "[concat(parameters('resourceNamePrefix'), '-wkr-as')]",
```

Comment:
Can we be consistent with these names, please? I vote to prefix with `worker`, rather than hiding it in the middle. Same applies to the deployment names.